bin/cron/sync_dropbox

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+# This syncs content changes from the shared dropbox folder to our github repo
+# on the staging machine. This is required for contents changes from dropbox to
+# be deployed.
+
+system("sudo apt-get install -y unison")
```

Comment:
Move this to a cookbook: https://codedotorg.atlassian.net/browse/STAR-991

Reply:
Here’s an example of a simple Chef cookbook that installs a single apt package:

https://github.com/code-dot-org/code-dot-org/blob/4f82855176b4d52b59aa29ee21ff9be3046bbcc4/cookbooks/cdo-java-7/recipes/default.rb#L8

Reply:
https://github.com/code-dot-org/code-dot-org/pull/33340